### ISSUE TITLE
Support for DCAT-AP 1.1

### ISSUE THREAD

**Issue by amercader** [ckan/ckanext-dcat#53]
Options:

* Update current default profile
* Create a new profile

**Comment by amercader** [comment]
Summary of changes:

**dcat:Dataset**

 - Add dct:accessRights
 - Add foaf:page (documentation)
 - Add dct:provenance
 - Add dct:type
 - Add dct:relation
 - Add dct:hasVersion (dataset)†
 - Add dct:isVersionOf (dataset)†
 - Add dct:source (dataset)†
 - Add admns:sample (distribution)†
 - Check admns:versionNotes (n) 
 - Check admns:identifier

† These link to other classes, we need to see if there is an easy way to store that

**dcat:Distribution**

 - Add spdx:checksum (algorithm + value)
 - Add foaf:page (documentation)
 - Add dct:language
 - Add dct:conformsTo (linked schemas, check work on Data Packages)
 - Add dct:modified
 - Check dct:description (n) 
 - Check dct:title (n)

**Comment by jakubklimek** [comment]
There are many issues with DCAT-AP v1.1 compatibility, mainly:

- literals instead of RDF resources
- missing multilingual support
- missing controlled vocabulary support
- no support in the CKAN Dataset and Resource forms

etc.

There is the <a href="http://www.europeandataportal.eu/en/content/providing-data/how-to-be-harvested-by-us">European Data Portal (EDP) DCAT-AP mapping</a> which is slightly incompatible with this extension and then there is the <a href="http://data.opendatasupport.eu:3030/dcat-ap_validator.html">DCAT-AP validator</a>.

- Are you going to merge with the EDP extension?
- Do you plan to support multilingual labels?
- Do you plan to support the European controlled vocabularies?
- Do you plan to extend the CKAN Dataset and Resource forms accordingly?

I ask so that it is clearer what features are planned or being developed by the CKAN team, which by the EDP team and which are not planned at all an can therefore be contributed by someone else.

**Comment by amercader** [comment]
The issues you mention are known:

> - literals instead of RDF resources 

See #50 (This should be easy to fix if you are up for a PR)

> - missing multilingual support

See #55

> - missing controlled vocabulary support
> - no support in the CKAN Dataset and Resource forms

See #56 

> There is the European Data Portal (EDP) DCAT-AP mapping which is slightly incompatible with this extension and then there is the DCAT-AP validator.

Thanks for pointing to these. Can you summarize some examples of why it's incompatible?

>    Are you going to merge with the EDP extension?

I'd love for that to happen (at least for the DCAT specific versions), but AFAIK they haven't yet released their extensions publicly so it's hard to assess how likely is that to happen

>    Do you plan to support multilingual labels?

Yes, see #55 

>    Do you plan to support the European controlled vocabularies?
>    Do you plan to extend the CKAN Dataset and Resource forms accordingly?

Yes, see #56 (I'm still not sure to which extent we would support the vocabularies). Note that this can be currently done by any instance by customizing their schemas to fit the CKAN/DCAT mapping. The scheming work will only make this easier out of the box.

> I ask so that it is clearer what features are planned or being developed by the CKAN team, which by the EDP team and which are not planned at all an can therefore be contributed by someone else.

As of now there is no funding to undertake major changes like #55 and #56. These two would be my priorities if there's time available in the next few months. If you know of some source of resources I'd be very happy to help with specs, etc. As for the EDP team we are currently not in contact, if you have a contact there feel free to point them here.

**Comment by akuckartz** [comment]
Mentioned this issue in https://joinup.ec.europa.eu/asset/ogd2_0/issue/dcat-ap#comment-18468

**Comment by amercader** [comment]
Thanks @akuckartz

**Comment by CaptSolo** [comment]
Re this question by @jakubklimek :
> Do you plan to support the European controlled vocabularies?
Do you plan to extend the CKAN Dataset and Resource forms accordingly?

CKAN-AP 1.1 requires to use European controlled vocabs (e.g. http://publications.europa.eu/mdr/authority/frequency ) for some properties. 

It would be great to have them supported in ckanext-dcat. If I understood @amercader correctly, implementers currently have to achieve this by customizing their CKAN instances locally.
